.github/workflows/build.yml

```diff
@@ -208,3 +208,15 @@ jobs:
           files: temp/coverage.xml
           flags: unittests
           fail_ci_if_error: true
+
+  typos-check:
+    name: "Typos"
+
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v3"
+
+      - name: "Search for misspellings"
+        uses: "crate-ci/typos@master"
```

Comment:
Is it possible to use v1 or something like this?

Reply:
No, they don't have this kind of tag.
https://github.com/crate-ci/typos/tags
Typos is getting better at typos fast.

BTW I find their GHA action terrible.
Using this instead https://github.com/szepeviktor/byte-level-care/blob/f1c750a0424ab8a0b7e96cd253f8cb3973b324c6/.github/workflows/spelling.yml#L31-L50

Reply:
Ok, so I would use the latest tag v1.16.17

Reply:
Done 🍏